src/components/tech/WorkingBox.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Badge,
+  Box,
+  Container,
+  Flex,
+  Heading,
+  Link,
+  Section,
+  Text,
+} from "@radix-ui/themes";
+import React from "react";
+
+interface WorkingBoxProps {
+  title: string;
+  para: string;
+  link: string;
+  tech: string;
+}
+
+const colors: { [key: number]: string } = {
+  0: "#3E63DD",
+  1: "#5AC597",
+  2: "#98BEC6",
+  3: "#DFCDE8",
+  4: "#4362A6",
+};
+
+export const WorkingBox = ({ title, para, link, tech }: WorkingBoxProps) => {
+  const data = tech.split(",");
+  return (
+    <Flex
+      gap="4"
+      align="start"
+      direction="column"
+      p="20px"
+      width="full"
+      justify="start"
+      maxWidth={{ md: "610px", initial: "auto" }}
+      className="bg-navy-50"
+    >
+      <Text size="7" weight="medium" className="text-navy-800">
+        {title}
+      </Text>
+      <Text size="4" className="leading-10">
+        {para}.<Link href={`https://${link}`}>[GitHub Repo]</Link>
+      </Text>
+
+      <Box>
+        {data.map((item, index) => {
+          return (
+            <Badge
+              key={index}
+              className={`${
+                index % Object.keys(colors).length === 0 ||
```

Comment:
**suggestion:** It'd be helpful to include a comment here to clarify the intent of this expression.  It seems like you're choosing to make the badge text color light or dark based on the background color?